src/pages/deconnexion/Deconnexion.tsx

```diff
@@ -9,11 +12,36 @@ export type QuestionnaireParams = {
 
 function Deconnexion(props: PortailProps) {
   const { survey } = useParams<QuestionnaireParams>();
-
+  useDocumentTitle("Page de déconnexion");
```

Comment:
Can you use a dictionary to enable multi-language ?

Reply:
in the todo list

Reply:
j'ajoute un ticket I18N dans notre backlog et merge la PR